lib/Comment/CommentBody.js

```diff
@@ -0,0 +1,70 @@
+import React, { Fragment } from 'react';
+import { arrayOf, bool, func, node, shape } from 'prop-types';
+import Linkify from 'linkifyjs/react';
+import CommentForm from '../Conversation/CommentForm';
+import Person from '../Person';
+
+export function CommentBody({
+  children,
+  isEditing,
+  comment,
+  users,
+  onEditChange,
+  onEditCancel,
+  onEditSubmit
+}) {
+  return (
+    <Fragment>
+      <Person person={comment.author} className="conversation" />
+      <div className="conversation__comment-body">
+        {!isEditing && (
+          <p className="conversation__text">
+            <Linkify>{comment.body}</Linkify>
```

Comment:
💚 we have a thing to highlight any mentions in the current version of Conversation, are you thinking of moving that over to the app side now?